strawberry_django/type.py

```diff
@@ -95,6 +95,16 @@ def _process_type(
     else:
         model_fields = []
 
+    # If MAP_AUTO_ID_AS_GLOBAL_ID is True, we can no longer set the id
+    # from fields or it will override the GlobalID and return the default
+    # django id instead in the query-result. This adjustment however still
+    # does not fix if the id was set to auto manually on the ModelType.
+    MAP_AUTO_ID_AS_GLOBAL_ID = django_settings().get("MAP_AUTO_ID_AS_GLOBAL_ID", False)
+    if MAP_AUTO_ID_AS_GLOBAL_ID:
```

Comment:
I personally don't like using UPPER_CASED for variable names other than constants.

I suggest either setting the variable lower cased, or better yet, use the settings directly like in my suggestion:

```suggestion
    if django_settings().get("MAP_AUTO_ID_AS_GLOBAL_ID", False):
```

Reply:
I can see that, it was more to indicate it came from the settings.  But fair enough.
What I wouldn't do it fetch it directly from the settings inside the if statement.  It makes for hard to read code.